### DISCUSSION TITLE
Cannot login (pycharm) [SOLVED]

### DISCUSSION THREAD

**Discussion by slvstr1** [community/community#34191] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/13185007/192266894-9868b453-fbfc-42ae-9952-7bec06f3de6a.png)

Using Pycharm, I am logged in in github, but it is not recognized (see picture above).

Not sure if this is a Pycharm or a GithubCop issue

**Comment by iamcjwilliams** [comment]
@slvstr1 The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted
